PWGMM/Mult/Tasks/heavy-ion-mult.cxx

```diff
@@ -244,6 +280,64 @@ struct HeavyIonMultiplicity {
     }
   }
   PROCESS_SWITCH(HeavyIonMultiplicity, processMC, "process MC", false);
+
+  void processDataCentFT0C(CollisionDataTableCentFT0C::iterator const& collision, FilTrackDataTable const& tracks)
+  {
+    bool hasCentrality = false;
```

Comment:
Rather then making it a runtime variable, you can make it `constexpr`, then you can use it in `if constexpr ()` conditions. So `constexpr auto hasCentrality = CollisionDataTableCentFT0C::template contains<aod::CentFT0Cs>();`